### PR TITLE
Admin commands on databases and graphs now support multiple databases

### PR DESCRIPTION
**Pull request by Lojjs**
Also fixed some multiple roles and users occurrences for consistency.

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc [hunk 1]
```diff
@@ -47,23 +47,23 @@ Databases created after this command execution will also be associated with thes
...
 | Grant a privilege to one or multiple roles.
 
 | [source, cypher]
-DENY database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname \| *}} TO role[, ...]
+DENY database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname [, ...] \| *}} TO role[, ...]
 | Deny a privilege to one or multiple roles.
 
 | [source, cypher]
-REVOKE GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname \| *}} FROM role[, ...]
+REVOKE GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname [, ...] \| *}} FROM role[, ...]
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc]: as above

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc [hunk 2]
```diff
@@ -47,23 +47,23 @@ Databases created after this command execution will also be associated with thes
 | Command | Description
 
 | [source, cypher]
-GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname \| *}} TO role[, ...]
+GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname [, ...] \| *}} TO role[, ...]
 | Grant a privilege to one or multiple roles.
 
 | [source, cypher]
-DENY database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname \| *}} TO role[, ...]
+DENY database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname [, ...] \| *}} TO role[, ...]
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc]: as above

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc [hunk 3]
```diff
@@ -47,23 +47,23 @@ Databases created after this command execution will also be associated with thes
 | Command | Description
 
 | [source, cypher]
-GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname \| *}} TO role[, ...]
+GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname [, ...] \| *}} TO role[, ...]
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc]: Is there a reason for using **dbname** here? The examples above all say just **name.**

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc [hunk 4]
```diff
@@ -47,23 +47,23 @@ Databases created after this command execution will also be associated with thes
...
 | Deny a privilege to one or multiple roles.
 
 | [source, cypher]
-REVOKE GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname \| *}} FROM role[, ...]
+REVOKE GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname [, ...] \| *}} FROM role[, ...]
 | Revoke a granted privilege from one or multiple roles.
 
 | [source, cypher]
-REVOKE DENY database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname \| *}} FROM role[, ...]
+REVOKE DENY database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname [, ...] \| *}} FROM role[, ...]
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc]: as above

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc [hunk 5]
```diff
@@ -47,23 +47,23 @@ Databases created after this command execution will also be associated with thes
...
 | Revoke a denied privilege from one or multiple roles.
 
 | [source, cypher]
-REVOKE database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname \| *}} FROM role[, ...]
+REVOKE database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname [, ...] \| *}} FROM role[, ...]
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc]: as above

**Review by AlexicaWright — COMMENTED** [review]
Please see my comment about **dbname** vs. **name**, but otherwise it looks good to me.

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc (hunk 3 above)
Comment (mnd999) [reply cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc]: That's just how it was, I will change it to make it consistant

**Comment by davidoliverSP2** [comment]
[Allow Multiple Databases in Database Privileges Commands](https://trello.com/c/l1C0PRig/82-allow-multiple-databases-in-database-privileges-commands)

**Pull request merged**